### ISSUE TITLE
'Block: Button' not using latest button structure

### ISSUE THREAD

**Issue by lewisself64** [WPTT/theme-unit-test#63]
The buttons blocks on the "Block: Button" post is out dated and should be wrapped inside of the `.wp-buttons` wrapper. If theme developers use this class, the theme may appear to have bugs.

**Comment by joyously** [comment]
Are you saying that someone who created a Button block when this markup was produced, would have to edit their post in order for it to get the current markup, so that the theme could show it correctly? 
Shouldn't the theme style the HTML so that all the variations(legacy, intermediate, and current) look correct?

**Comment by lewisself64** [comment]
Yes that's what I meant. I have modified my theme to style it without this class and I did wonder if it was purposely like this to try to make sure the button styling is backwards compatible but if that is the case it wasn't clear and I thought I had a bug. Would it not be a good idea to create a new section with the more current buttons and title it to make it obvious to the developer?
